src/app/modules/game/services/game-view.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject} from "rxjs";
+
+@Injectable({
+  providedIn: "root",
+})
+export class GameViewService {
+  private readonly _currentView = new BehaviorSubject<string>('');
+  readonly currentView$ = this._currentView.asObservable();
+
+  get currentView(): string {
+    return this._currentView.getValue();
+  }
+  set currentView(view: string) {
```

Comment:
2 methods with same name but different login. Feel little confused )

Reply:
one's a set, another is a get :/